checks/collector.py

```diff
@@ -205,7 +205,8 @@ def __init__(self, agentConfig, emitters, systemStats, hostname):
             'memory': u.Memory(log),
             'processes': u.Processes(log),
             'cpu': u.Cpu(log),
-            'system': u.System(log)
+            'system': u.System(log),
+            'fh': u.File_Handlers(log)
```

Comment:
just for the sake of having a clear log message if an error occurs during the check run, could we name the key `file_handles` instead?